electron/locale/strings.js

```diff
@@ -49,6 +49,7 @@ string.menuQuit = 'Quit Wire';
 string.menuShowHide = 'Show/Hide Menu';
 string.menuSavePictureAs = 'Save Picture As...';
 string.menuNoSuggestions = 'No Suggestions';
+string.menuAutoStart = 'Start with OS';
```

Comment:
Since we have `string.menuStartup = 'Launch Wire on Windows Start';` maybe we can just remove Windows from that string and use the same for all.